src/main/java/net/sf/jabref/importer/fetcher/DOItoBibTeXFetcher.java

```diff
@@ -86,14 +85,18 @@ public JPanel getOptionsPanel() {
         return null;
     }
 
-    public BibEntry getEntryFromDOI(String doiStr, ParserResult parserResult) {
-        Objects.requireNonNull(parserResult);
+    public BibEntry getEntryFromDOI(String doiStr) {
+        return getEntryFromDOI(doiStr, null);
```

Comment:
Some places just don't display error but solely use the fetcher. previously only null was passed for the ParserResult. I thought overloading the method makes more sense, but we can just delete the overloaded method again.